packages/block-editor/src/components/rich-text/index.js

```diff
@@ -463,7 +463,7 @@ RichTextContainer.Content = ( { value, tagName: Tag, multiline, ...props } ) =>
 	const content = <RawHTML>{ html }</RawHTML>;
 
 	if ( Tag ) {
-		return <Tag { ...omit( props, [ 'format' ] ) }>{ content }</Tag>;
+		return <Tag className={ className || undefined } { ...omit( props, [ 'format' ] ) }>{ content }</Tag>;
```

Comment:
@ellatrix - this is necessary to avoid the case where `<ul class="">...</ul>` is saved in the post content. Does it make sense?

Reply:
Why does the parent not pass `undefined`? What value is it passing?

Reply:
it passes an empty string: 
```js
classnames() === '';
```

Reply:
Ok, but then it should be handled by the parent, as the parent uses `classnames`, not `RichText`? E.g. `classnames() || null`.

Reply:
Yes, this is one approach but then all blocks need to do it which is suboptimal. I prefer we have it covered. 

I can add this logic to the List block if you strongly disagree.